plan_parser.py

```diff
@@ -0,0 +1,44 @@
+import graph_utils
+import relops
+from data_params import DataParams
+
+
+def get_relop(rel):
+    relop_name = rel['relOp']
+    if relop_name == 'LogicalTableScan':
+        table_data = DataParams(100)
```

Comment:
Will need to customize it after the tree build as we'll want to reuse plans for different data